### PR TITLE
Update crate versions to 5.0.0-pre0

### PR DESCRIPTION
**Pull request by nick-mobilecoin**
### Future Work

[Soundtrack of this PR]()

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2511,7 +2511,7 @@ dependencies = [
 
 [[package]]
 name = "mc-attest-verifier-config"
-version = "4.0.2"
+version = "5.0.0-pre0"
```
Comment (nick-mobilecoin) [inline Cargo.lock:2514]: Note the larger bump

File: Cargo.lock [hunk 2]
```diff
@@ -3033,7 +3033,7 @@ dependencies = [
 
 [[package]]
 name = "mc-consensus-tool"
-version = "2.0.0"
+version = "5.0.0-pre0"
```
Comment (nick-mobilecoin) [inline Cargo.lock:3036]: Note the larger bump. Think this got missed a while back

File: Cargo.lock [hunk 3]
```diff
@@ -4665,7 +4665,7 @@ dependencies = [
 
 [[package]]
 name = "mc-fog-view-server-test-utils"
-version = "2.0.0"
+version = "5.0.0-pre0"
```
Comment (nick-mobilecoin) [inline Cargo.lock:4668]: Note the larger bump. Think this got missed a while back

File: attest/verifier/config/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mc-attest-verifier-config"
-version = "4.0.2"
+version = "5.0.0-pre0"
```
Comment (nick-mobilecoin) [inline attest/verifier/config/Cargo.toml:3]: Note the larger bump. Think this got missed a while back

File: consensus/tool/Cargo.toml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mc-consensus-tool"
-version = "2.0.0"
+version = "5.0.0-pre0"
```
Comment (nick-mobilecoin) [inline consensus/tool/Cargo.toml:3]: Note the larger bump. Think this got missed a while back

File: core/types/Cargo.toml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mc-core-types"
-version = "2.0.0"
+version = "5.0.0-pre0"
```
Comment (nick-mobilecoin) [inline core/types/Cargo.toml:3]: Note the larger bump. Think this got missed a while back

File: fog/view/server/test-utils/Cargo.toml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mc-fog-view-server-test-utils"
-version = "2.0.0"
+version = "5.0.0-pre0"
```
Comment (nick-mobilecoin) [inline fog/view/server/test-utils/Cargo.toml:3]: Note the larger bump. Think this got missed a while back

**Review by varsha888 — APPROVED** [review]

**Review by samdealy — APPROVED** [review]

**Pull request merged**